java-frontend/src/main/java/org/sonar/java/cfg/CFG.java

```diff
@@ -903,6 +902,13 @@ private void handleExceptionalPaths(Symbol symbol) {
   }
 
   private void buildTypeCast(Tree tree) {
+    enclosingTry.peek().catches.entrySet().stream()
+      .filter(e -> e.getKey().isSubtypeOf("java.lang.ClassCastException"))
+      .findFirst()
+      .ifPresent(e -> {
+      currentBlock = createBlock(currentBlock);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)